cmd/dockerd/daemon_windows.go

```diff
@@ -93,7 +94,30 @@ func newCgroupParent(config *config.Config) string {
 	return ""
 }
 
-func (cli *DaemonCli) initContainerD(_ context.Context) (func(time.Duration) error, error) {
+func (cli *DaemonCli) initContainerD(ctx context.Context) (func(time.Duration) error, error) {
+	if cli.Config.ContainerdAddr == "" && cli.Config.DefaultRuntime == config.WindowsV2RuntimeName {
+		cli.Config.ContainerdAddr = supervisor.GrpcPipeName
+		system.InitContainerdRuntime(cli.Config.ContainerdAddr)
+
+		var waitForShutdown func(time.Duration) error
+		logrus.Debug("Starting daemon managed containerd")
+		opts, err := cli.getContainerdDaemonOpts()
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to generate containerd options")
+		}
+
+		r, err := supervisor.Start(ctx, filepath.Join(cli.Config.Root, "containerd"), filepath.Join(cli.Config.ExecRoot, "containerd"), opts...)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to start containerd")
+		}
+		logrus.Debug("Started daemon managed containerd")
```

Comment:
While working on https://github.com/moby/moby/pull/43945, I noticed that this log was redundant (`remote. startContainerd()` already logs when it's started, including `PID`, and `address`), so this line can be removed.